Client/qtTeamTalk/languages/Russian.ts

```diff
@@ -86,17 +87,17 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../aboutdlg.cpp" line="34"/>
         <source>Compiled on </source>
-        <translation>Составленный на </translation>
+        <translation>Скомпелировано на </translation>
     </message>
     <message>
         <location filename="../aboutdlg.cpp" line="35"/>
         <source>Version </source>
-        <translation>Версия </translation>
+        <translation type="unfinished">Версия </translation>
```

Comment:
Several of the new items are listed as "unfinished". Is that intentional?

Reply:
Corrected the finish lines mark